src/parser/shared/modules/StatTracker.ts

```diff
@@ -671,16 +671,16 @@ class StatTracker extends Analyzer {
       if (returnRatingForNextPercent) {
         //Returns the rating needed for 1% at current rating levels
         return (
-          (baselineRatingPerPercent / (1 - penaltyThresholds[idx - 1].penaltyAboveThis) / coef) *
+          (baselineRatingPerPercent / (1 - (penaltyThresholds[idx - 1].penaltyAboveThis || 0)) / coef) *
```

Comment:
Negative index crashes this. I think that at index of 0, it should be 0, but I don't know enough about this module to know if this will break things.

Reply:
How are you getting negative index here

Reply:
Also to be clear, this function cannot be used at all for anything wotlk

Reply:
The only way to get a negative index there is if a player had below 0% of a given secondary/tertiary from rating, which shouldn't be possible

Reply:
It was crashing on this log: 

https://dragonflight.wowanalyzer.com/report/Ntvc2rAQ3TFL4k1K/56-Normal+Sartharion+-+Kill+(5:41)/Tolario/disc/character

Looking at the code, I'm not sure exactly what the root cause was.

I'll work on making this page more WOTLK friendly when I get a chance.

Reply:
If wotlk logs are using the stattracker they will never be accurate. That is strictly tied to retail stats and using the diminishing returns sytem in place from Shadowlands and onwards

Reply:
Is your formatter setup differently than the projects? even if I force format the file, I don't get any changes